src/test/java/teammates/test/cases/ui/InstructorEditStudentFeedbackSaveActionTest.java

```diff
@@ -219,17 +212,17 @@ private void testIncorrectParameters() throws Exception {
                     Const.ParamsNames.COURSE_ID), e.getMessage());
         }
         
-        
         ______TS("Unsuccessful case: test no moderated student parameter");
+        
         submissionParams = new String[]{
-                Const.ParamsNames.FEEDBACK_QUESTION_RESPONSETOTAL + "-2", "1",
-                Const.ParamsNames.FEEDBACK_RESPONSE_ID + "-2-0", fr.getId(),
+                Const.ParamsNames.FEEDBACK_QUESTION_RESPONSETOTAL + "-1", "1",
```

Comment:
Everything else in here is 1, not sure why this is 2 but it doesn't seem to break anything changing them to 1